### ISSUE TITLE
TESS content issues

### ISSUE THREAD

**Issue by srallen** [zooniverse/front-end-monorepo#848]
This is from feedback from the internal review of the TESS beta project on production:

- Field guide:
  - [ ] Is it possible to include text links in the Field Guide?  Just a way to get to a certain spot without scrolling for a Long Time. Note: Markdownz supports table of contents, but this might not work in the current modal display.
  - [ ] There are lots of images and they’re all quite large, too, which makes it a bit difficult to process the info. 
  - [ ] Should the images be updated to look like the current LCV?
- Feedback
  - [ ] Feedback appears to repeat the same info unnecessarily w/ many bullets. This is fixed by removing the messages set for success and failure from the subject metadata per subject, then the UI will fallback to the default messages.
- Talk
  - [ ] The JSON data being encoded as text makes the Talk pages impossible to use. This is fixed by making the image the first location during subject upload.

**Comment by beckyrother** [comment]
Field guide: https://github.com/zooniverse/front-end-monorepo/issues/856

**Comment by eatyourgreens** [comment]
>Feedback appears to repeat the same info unnecessarily w/ many bullets. This is fixed by removing the messages set for success and failure from the subject metadata per subject, then the UI will fallback to the default messages.

My understanding of feedback is that this will still show a list with the default message for each rule. The reducer returns an array of messages, one for each rule. https://github.com/zooniverse/front-end-monorepo/blob/59c3ec128b2ba4752c58853e4dccbb2f0f17eff8/packages/lib-classifier/src/store/FeedbackStore.js#L26-L29

**Comment by srallen** [comment]
Yes, you're correct that it'll show the message set per rule, so in this case there will still be up to 13. At that point we can reasonably implement the reducer idea you had. If there's a message in the subject metadata, though, we have to assume they want it shown since it was intended for custom messages per subject.

**Comment by srallen** [comment]
The many bullets with repeated messages will remain and issue until we decide how to handle that. I've noted this in #836

**Comment by rogerhutchings** [comment]
Noting that @camallen suggested removing the feedback messages entirely to solve this

**Comment by srallen** [comment]
I don't agree with that as it will make it appear that there's been a regression in functionality when we switch over. Also, I think volunteers would be upset if they don't get any contextual information about what the green or red columns mean. 

The bulleted list is an annoyance but not enough of a one to hide it. If someone complains, we can let them know we're working on a new UI design for this, which we will be.
